config/application.rb

```diff
@@ -31,14 +31,12 @@ class Application < Rails::Application
         'Access-Control-Allow-Methods' => 'POST, PUT, DELETE, GET, OPTIONS',
         'Access-Control-Allow-Headers' => 'Origin, X-Requested-With, Content-Type, Accept, Authorization, tgtoken, tgToken, bearer-token',
         'Content-Type' => 'application/json'
-
-    #config.action_dispatch.default_headers.merge!('Content-Type' => 'application/json')
-
     }
 
     config.autoload_paths += Dir["#{config.root}/app/models/", "#{config.root}/lib/**/"]
 
     config.iiif_collections_host = ENV['IIIF_COLLECTIONS_HOST']
     config.s3_download_prefix = ENV['S3_PUBLIC_DOWNLOAD_PREFIX']
+    config.use_jwt_auth = ENV['USE_JWT_AUTH'] == 'Y'
```

Comment:
If this is set to 'N', wont the evaluation in check_anno_auth still be true? Would (if possible) using boolean values be better here? Maybe some logic to convert 'Y' and 'N' to boolean values?

Reply:
config.use_jwt_auth will evaluate to true if the env var is 'Y', otherwise it will always be false, because the "==" operator always returns a boolean.

Reply:
`config.use_jwt_auth` should be false for any value of `USE_JWT_AUTH` besides `Y`, so the else block should activate in chec_anno_auth.

```
irb(main):001:0> test = 'y' == 'n'
=> false
irb(main):002:0> test = 'y' == 'y'
=> true
```

Reply:
Sorry, read that as default assignment. This should fine.

Reply:
Maybe just parenthesis around the expression just for readability.

Reply:
Added parens